<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisWdgHalftoneGen</class>
 <widget class="QWidget" name="KisWdgHalftoneGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KisColorButton" name="bnForeColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KoColorSlider" name="sldIntensity" native="true"/>
     </item>
     <item>
      <widget class="KisColorButton" name="bnBackColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Colors:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="spAngle"/>
     </item>
     <item>
      <widget class="QDial" name="dialAngle"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkAntiAliasing">
     <property name="text">
      <string>Anti-Aliasing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KisSliderSpinBox" name="sldSize" native="true"/>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkInvert">
     <property name="text">
      <string>Invert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoColorSlider</class>
   <extends>QWidget</extends>
   <header>KoColorSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
